myapp/src/components/Thread/Form/index.js

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react'
+import {CommentInput} from './FormElements.js';
+
+function Form(props) {
+  const [person, setPerson] = useState(
+    {
+      user: "",
+      post: ""
+    }
+  );
+
+  function handleChange(event) {
+      const value = event.target.value;
+      setPerson({
+        ...person,
+        [event.target.name]: value
+      });
+  }
+
+  function submitForm() {
+      props.handleSubmit(person);
+      setPerson(
+        {user: "", post: ""}
+      );
+  }
+
+  return (
+    <>
+      <form>
```

Comment:
The same comment as the other file, you can remove `<> </>` since only one component is being returned